.baseline/copyright/999_palantir.txt

```diff
@@ -0,0 +1 @@
+(c) Copyright ${today.year} Palantir Technologies Inc. All rights reserved.
```

Comment:
file shouldn't exist since it makes the palantir internal copyright the default one.

Reply:
removed

Reply:
do we have any other standard configurations for things like this other than `internal` and `external`? would be nice to flag this once since this file will come back every time we `./gradle updateBaselineConfig`

Reply:
I think we want things to default to the internal copyright.

Reply:
Agree that internal should be the default. Just want to be able to make a one time change so I don't screw up copyrights every time I run the `updateBaselineConfig` task